src/tools/tests/schema2kotlin-test.ts

```diff
@@ -174,7 +174,7 @@ describe('schema2kotlin', () => {
       `particle P
         h1: reads Person {name: Text}
       `, [
-        'typealias P_H1 = AbstractP.P_H1'
+        'typealias P_H1 = AbstractP.Person'
```

Comment:
Please add tests covering schema-name based aliases and non schema-name based ones, e.g.:

```
particle P
  h1: reads Person {name: Text}
  h2: reads Person {age: Number}
```

We could also add this one here for completeness:
```
particle P
  h1: reads * {name: Text}
```

Reply:
Added two tests for these cases.